website/sidebars.json

```diff
@@ -21,7 +21,8 @@
           "build-pdk",
           "build-cumulus",
           "build-smart-contracts",
-          "build-oracle"
+          "build-oracle",
+          "build-wallets"
         ]
       },
       {
```

Comment:
Please also run "prettier" again after changes.

Mo has also reviewed and is OK with the basic structure.